wandb/integration/monai/model_checkpoint.py

```diff
@@ -0,0 +1,107 @@
+from pathlib import Path
+from typing import Any, Callable, Mapping, Optional, Union
+
+from ignite.engine import Engine
+
+import wandb
+from wandb.util import get_module
+
+ignite_engine = get_module("ignite.engine")
+ignite_handlers = get_module("ignite.handlers")
```

Comment:
what happens if this is not available, i think this code will fail. Try running this without ignite installed. 
Also why do you have on line 4 direct import and on line 9 you use get_module?